<ui version="4.0" >
 <class>ScanWidget</class>
 <widget class="QWidget" name="ScanWidget" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>266</width>
    <height>273</height>
   </rect>
  </property>
  <property name="sizePolicy" >
   <sizepolicy vsizetype="Fixed" hsizetype="Fixed" >
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_3" >
   <item>
    <layout class="QGridLayout" name="gridLayout" >
     <item row="0" column="0" >
      <layout class="QVBoxLayout" name="verticalLayout" >
       <item>
        <widget class="QLabel" name="m_interfaceLabel" >
         <property name="text" >
          <string>Interface</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="KComboBox" name="m_interface" />
       </item>
      </layout>
     </item>
     <item row="0" column="1" >
      <layout class="QVBoxLayout" name="verticalLayout_2" >
       <item>
        <widget class="QLabel" name="m_viewLabel_2" >
         <property name="text" >
          <string>View</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="KComboBox" name="m_view" >
         <item>
          <property name="text" >
           <string>Map</string>
          </property>
         </item>
         <item>
          <property name="text" >
           <string>Details</string>
          </property>
         </item>
        </widget>
       </item>
      </layout>
     </item>
     <item row="1" column="0" colspan="2" >
      <widget class="QTableView" name="m_detailsView" >
       <property name="enabled" >
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QWidget" native="1" name="m_scanView" />
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>KComboBox</class>
   <extends>QComboBox</extends>
   <header>kcombobox.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
